docs/guides/compiling-typescript-for-deployment.md

```diff
@@ -41,3 +41,26 @@ Rememeber to set `PLT_TYPESCRIPT=false` in your environment variables in the dep
 
 If you are building a [Runtime](/reference/runtime/introduction.md)-based application, you will need
 to compile every service independently or use the `plt runtime compile` command.
+
+## Avoid shipping the TypeScript sources
+
+If you want to avoid shipping the TypeScript sources you need to configure Platformatic the location
+where your files have been built, adding an `outDir` option:
```

Comment:
```suggestion
where your files have been built by adding an `outDir` option:
```

Reply:
Correct me if I'm wrong here, but It also works to ship your `dist/` folder, keep your `tsconfig.json` but no `*.ts` sources. The `outDir` is only needed if you _don't_ want to ship your `tsconfig.json`